### DISCUSSION TITLE
Want to make much money? :\

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#231] [Feature Requests & Ideas]
Hi, I just found this CMS. Why don't make it total FREE if you open full sourse code? You can make money like Wordpress.com they open full code at Wordpress.org. 
I will try your CMS, it look like well :) Thank you for made this.

**Comment by jmikrut** [comment]
Hey @IRediTOTO — great question. We've very deliberately built out our pricing model to ensure that our team can not only build, but successfully maintain Payload well into it becoming the absolute best choice for modern dev teams when building new sites and apps.  

We think that its demonstrated value greatly offsets the low price that we are charging, which will in turn allow developers like yourself to pass off the cost of the CMS to your client, while at the same time ensure that Payload's future is stable and brilliant. Definitely not about making a lot of money... Moreso, my team and I want to be able to devote ourselves full-time to a product that we very much so believe in.

What do you think about our pricing model? Even though it is not totally free, do you think that our prices are fair?

**Reply by IRediTOTO** [reply]
Price is fair for team who made money, Cheap i think. But open full for everyone  will make you growth faster i think. You can sell your hosting service and free will make you famous. I like your pack, will use it soon :) so powerful. I tried many Nodejs CMS but i feel this is best for now. Good UI and easy api.
